src/advection_stencils.jl

```diff
@@ -1,4 +1,4 @@
-export l94_stencil, ppm_stencil, upwind1_stencil, upwind2_stencil
+export l94_stencil, ppm_stencil, upwind1_stencil, upwind2_stencil, ppm_stencil_grid
```

Comment:
I don't think we have a `ppm_stencil_grid` anymore, so that entry here can be deleted.